ui/app/src/components/Search/Search.tsx

```diff
@@ -26,18 +26,22 @@ import { debounceTime, distinctUntilChanged } from 'rxjs/operators';
 import { reversePluckProps } from '../../utils/object';
 import LookupTable from '../../models/LookupTable';
 import { UNDEFINED } from '../../utils/constants';
+import useMount from '../../hooks/useMount';
 
 export function Search(props: SearchProps) {
-  const { mode = 'default', onSelect, embedded = false, onAcceptSelection, onClose } = props;
+  const { mode = 'default', onSelect, embedded = false, onAcceptSelection, onClose, parameters } = props;
 
   // region State
   const [filters, setFilters] = useSpreadState({
-    sortBy: undefined,
-    sortOrder: undefined
+    sortBy: parameters?.sortBy ?? undefined,
+    sortOrder: parameters?.sortOrder ?? undefined
```

Comment:
I do wonder if this is even necessary. If these props aren't supplied, they will already be undefined. If only the sub-props are not supplied, they will also be undefined.
 ```suggestion
    sortBy: parameters?.sortBy ?? UNDEFINED,
    sortOrder: parameters?.sortOrder ?? UNDEFINED
```